### DISCUSSION TITLE
I can't seem to style buttons fully

### DISCUSSION THREAD

**Discussion by Schmell** [wobsoriano/svelte-sonner#81] [Q&A] [answered]
I have tried, but the buttons do not seem to get styled from the `toastOptions`,

By the way, I am using daisyUI for these classes. but I have tried regular tailwind and  my own classes
```svelte
<Toaster
    closeButton
    theme="dark"
    toastOptions={{
    unstyled: true,
    classes: {
        actionButton: 'bg-primary',
        cancelButton: 'bg-error',
        closeButton: 'text-error',
        toast: 'flex items-center gap-2 bg-base-300 p-4 rounded-lg',
        success: 'bg-success text-success-content',
        error: 'bg-error text-error-content',
        warning: 'bg-warning text-warning-content',
        info: 'bg-info text-info-content',
        description: 'text-base-content'
    }
}}
/>
```
The toast, success, error etc.. work as expected
Confusingly I can add padding or margin and even `font-bold`, but cannot add a background colour, change the text-size, add a border, shadow etc..

Thank you in advance for any help you can provide
Schmell

**Comment by Schmell** [comment]
Is this repo dead?? 
No one seems to be responding to anyone.

**Reply by wobsoriano** [reply]
No. I have full time job but I'll defo check stuff out as soon as I can.

**Comment by wobsoriano** [comment, marked as answer]
Apologies for the delayed response. There was an issue with the action button styling persisting even when the `unstyled` prop was set to `true`.

Fixed in https://github.com/wobsoriano/svelte-sonner/pull/88

**Reply by Schmell** [reply]
Sorry about the previous comment.
Thanks for the fix .
